app/routes.py

```diff
@@ -1,2 +1,101 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, request
 
+planets_bp = Blueprint("planets_bp", __name__, url_prefix="/planets")
+
+
+@planets_bp.route("", methods=["POST", "GET"])
+def handle_planets():
+    if request.method == "POST":
+        request_body = request.get_json()
+        if "name" not in request_body or "description" not in request_body:
+            return jsonify("Invalid request"), 404
+
+        new_planet = Planet(
+            name=request_body["name"],
+            description=request_body["description"]
+        )
+        db.session.add(new_planet)
+        db.session.commit()
+
+        return jsonify(f"Planet {new_planet.name} created successfully"), 201
+
+    elif request.method == "GET":
+        name_query = request.args.get("name")
+        if name_query:
+            planets = Planet.query.filter_by(name=name_query)
+        else:
+            planets = Planet.query.all()
+
+        planets_response = []
+        for planet in planets:
+            planets_response.append({
+                "id": planet.id,
+                "name": planet.name,
+                "description": planet.description
+            })
```

Comment:
We strongly suggest making a helper method on the Planet class to take care of converting the model instance to a dictionary for the purpose of being returned.